test/e2e/scheduling/predicates.go

```diff
@@ -1045,23 +1045,20 @@ func translateIPv4ToIPv6(ip string) string {
 // GetPodsScheduled returns a number of currently scheduled and not scheduled Pods on worker nodes.
 func GetPodsScheduled(workerNodes sets.String, pods *v1.PodList) (scheduledPods, notScheduledPods []v1.Pod) {
 	for _, pod := range pods.Items {
-		if workerNodes.Has(pod.Spec.NodeName) {
```

Comment:
Previously this was `if !masterNodes.Has(pod.Spec.NodeName)` so this isn't fully going back to the same functionality. The check for unscheduled nodes could consider master nodes in this case, but from my understanding of this check that would be fine.

Reply:
Very reasonable change for me.
Why we checked the NodeName is in names of workerNodes before checking the NodeName is empty (^^;)